.github/labeler.yml

```diff
@@ -128,6 +128,11 @@ RZIL:
   - librz/il/**/*
   - librz/core/cil.c
 
+FLIRT:
+  - librz/flirt/*
```

Comment:
Please fix this as well then. Also, add or update the `signature` label as well.